### ISSUE TITLE
Increase contrast of adjacent rainbow parens colors

### ISSUE THREAD

**Issue by cap10morgan** [cursive-ide/cursive#2263]
Currently many of the adjacent rainbow parens colors are quite similar to each other, and thus difficult to visually determine at a glance. It would increase usability of the feature to put higher contrast colors next to each other.

So, for example, instead of this:

<img width="37" alt="Screen Shot 2019-11-06 at 9 21 59 AM" src="https://user-images.githubusercontent.com/59748/68316622-fe407700-0076-11ea-9135-bd696a89b54b.png">

Something more like this (taken from Calva in VS Code):

<img width="37" alt="Screen Shot 2019-11-06 at 9 22 14 AM" src="https://user-images.githubusercontent.com/59748/68316682-1912eb80-0077-11ea-82c4-2039249ea443.png">

In the first screenshot, the colors of adjacent parens are also relatively adjacent in the color spectrum and are thus difficult to visually differentiate. In the second screenshot, adjacent parens are drawn with colors that are further apart in the spectrum and thus easier to visually differentiate.

**Comment by cursive-ide** [comment]
You can actually customise these colours at _Preferences | Editor | Color Scheme | Clojure_. If you have an alternative scheme that you think looks better, I'm happy to consider including it in Cursive by default. I'm not sure what the Calva licence is, perhaps I could just use their colour scheme if they don't mind.

**Comment by cap10morgan** [comment]
Oh great, thanks! Looks like Calva is MIT: https://github.com/BetterThanTomorrow/calva/blob/master/LICENSE

**Comment by cap10morgan** [comment]
And looks like these are the colors: https://github.com/BetterThanTomorrow/calva/blob/870f07ee89d37d6aec6db5881ce997a48ccf30be/src/highlight/src/extension.ts#L113

**Comment by cap10morgan** [comment]
<img width="131" alt="Screen Shot 2019-11-06 at 2 38 12 PM" src="https://user-images.githubusercontent.com/59748/68340141-18904a00-00a3-11ea-9c35-edd4cf4a657c.png">

That's Calva's colors with your deeper levels left intact (which I think actually look pretty good already).

**Comment by cursive-ide** [comment]
Thanks! Could you export that (from the drop-down menu on the gear icon) and email it to me at cursive@cursive-ide.com?

Thanks for doing that, I don't use rainbow brackets and I'm also colourblind, so I'm not the ideal person to be making these choices :)

**Comment by cap10morgan** [comment]
Sure thing

**Comment by cap10morgan** [comment]
@cursive-ide Did you receive the email? Was that the correct export and format?

**Comment by cursive-ide** [comment]
@cap10morgan Yes, I did - thank you! I haven't had a chance to look at it yet, I'll try to check it out today.
